libs/libopusenc/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2014-2016 OpenWrt.org
```

Comment:
drop this entirely, or use your own copyright, with a correct date.  (Unless you _actually_ have copyright assignment with openwrt.org on file)

Reply:
ok, gonna provide it properly.